react/week3/todo3/src/Components/Counter.js

```diff
@@ -0,0 +1,18 @@
+import React, {useState, useEffect} from 'react'
+
+export default ({ name }: Props) => {
```

Comment:
It looks like you are trying to add a type here, but this is not a TypeScript-file.

Reply:
React might complain that this component is missing a `componentName`.
This is because you're exporting an anonymous function as default.

I'd write 
```
const Counter = ({name}) => {
 ...
}

export default Counter
```